CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v5.2.0](https://github.com/voxpupuli/puppet-gitlab_ci_runner/tree/v5.2.0) (2024-07-18)
```

Comment:
please don't update the CHANGELOG.md. We will do this automatically during the next release.